solution/zenkoversion.yaml

```diff
@@ -97,6 +100,9 @@ spec:
       monitoring:
         image: jmx-javaagent
         tag: '${JMX_JAVAAGENT_TAG}'
+      cleaner:
+        image: kafka-cleaner-dev
```

Comment:
```suggestion
        image: kafka-cleaner
```

Reply:
thats done, was waiting for the release of the component